keyboards/planck/keymaps/deft/keymap.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * The Deft layout for the Planck Keyboard
+ * Version 1.0
+ *
+ * A keyboard layout deisgned for effeciency in programming and transferability between ortholinear boards.
```

Comment:
Spelling corrections.

```suggestion
 * A keyboard layout designed for efficiency in programming and transferability between ortholinear boards.
```